### ISSUE TITLE
[request] Add worker recommendation enabled as an option

### ISSUE THREAD

**Issue by smirea** [chaorace/cqui#107]
I actually found the worker recommendations useful, especially in the early stages of the game when you don't have all the tech to improve every tile. It helps show the immediate things you could do

I saw that this is disabled by default now, is it possible to add an option in the menu to re-enable it?

**Comment by chaorace** [comment]
I'm gonna consider this a dupe of #51. This is a coming feature, but all option additions are on hold for now while I wait on a pr to go through
